tests/testthat/test-cloned-repo.R

```diff
@@ -1,54 +0,0 @@
-## Check out the repo from github and run check in root directory to test.
```

Comment:
I removed this test because we use codemetar on codemetar's clone repo all the time anyway, so didn't seem worth having this test?